aeron-client/src/main/java/io/aeron/ChannelUriStringBuilder.java

```diff
@@ -2267,6 +2277,7 @@ public String build()
         appendParameter(sb, UNTETHERED_WINDOW_LIMIT_TIMEOUT_PARAM_NAME, untetheredWindowLimitTimeoutNs);
         appendParameter(sb, UNTETHERED_RESTING_TIMEOUT_PARAM_NAME, untetheredRestingTimeoutNs);
         appendParameter(sb, MAX_RESEND_PARAM_NAME, maxResend);
+        appendParameter(sb, "stream-id", streamId);
```

Comment:
Create a constant `STREAM_ID_PARAM_NAME` in the `CommonContext` class and use here.

Reply:
I had seconds thoughts about because it isn't an official parameter.

Reply:
It is (see above).

Reply:
Fixed